docs/audits/deprecated-api-usage.md

```diff
@@ -0,0 +1,17 @@
+# Avoid using deprecated GPT APIs
+
+## Overview
+
+This audit checks for the presence of warning and error messages coming from GPT which relate to deprecated API usage.
```

Comment:
This could be more direct: "This audit checks for deprecated API usage." 

Or maybe: This audit checks if the page invokes any deprecated GPT APIs.

Reply:
+1, how about "This audit ensures that deprecated Google Publisher Tag (GPT) APIs are not used."

Reply:
Got it, added your suggestion in.